publish.sbt

```diff
@@ -46,3 +46,18 @@ ThisBuild / description := "Login service for JWT public signing services"
 ThisBuild / organizationName := "ABSA Group Limited"
 ThisBuild / startYear := Some(2023)
 ThisBuild / licenses += "Apache-2.0" -> url("https://www.apache.org/licenses/LICENSE-2.0.txt")
+
+ThisBuild / pomIncludeRepository := { _ => false }
+ThisBuild / publishTo := {
+  val nexus = "https://oss.sonatype.org/"
+  if (isSnapshot.value) {
+    Some("snapshots" at s"${nexus}content/repositories/snapshots")
+  } else {
+    Some("releases" at s"${nexus}service/local/staging/deploy/maven2")
+  }
+}
+ThisBuild / publishMavenStyle := true
+
+ThisBuild / credentials += Credentials("Sonatype Nexus Repository Manager", "oss.sonatype.org", "TheLydonKing", "JamesBue10368Sonatype!!!")
```

Comment:
I don't think this the way to have a password there, @TheLydonKing  On local, it is should be solved by having the secret in .m2 settings.

@hamc17, do you know how to do this if we wanted/needed to push from the runner? Probably load secrets from Github secrets. But it would be ok to release form local only, too

Reply:
Oh whoops, did that by accident when publishing, was meant to remove that